tasks/95_pairwise/pairwise.js

```diff
@@ -0,0 +1,17 @@
+export const pairwise = (arr, num) => {
+  if (!arr.length) {
+    return 0;
+  }
+  let sumOfIndex = 0;
+  for (let i = 0; i < arr.length; i++) {
+    let index = arr.findIndex((cur, index) => {
+      return arr[i] + cur === num && index !== i;
```

Comment:
you can use just a for loop here and start from current index + 1, in this case you will not have to check for index !== i and you will not iterate array every iteration of loop above